src/sphinx/code/SqlToSlick.scala

```diff
@@ -43,7 +43,7 @@ object SqlToSlick extends App {
   val db = Database.forConfig("h2mem1")
   try {
 
-    Await.result(db.run(Action.seq(
+    Await.result(db.run(DBIO.seq(
```

Comment:
`seq` was not planned initially. I added it later, first as a method in AsyncTest and then directly in DBIO, because it's so ubiquitous.